Goal: Fix documentation links

Fixes: #508388

Status wrt upstream: Reported as #5968

Note: 

Index: samba-3.2.5/docs/htmldocs/index.html
===================================================================
--- samba/docs/htmldocs/index.html	2008-11-27 16:48:42.000000000 +0100
+++ samba.orig/docs/htmldocs/index.html	2008-12-14 08:40:14.841582235 +0100
@@ -23,19 +23,15 @@
  <td valign="top">This book provides example configurations, it documents key aspects of Microsoft Windows networking, provides in-depth insight into the important configuration of Samba-3, and helps to put all of these into a useful framework.</td>
 </tr>
 <tr>
- <td valign="top"><a href="../using_samba/toc.html">Using Samba</a>, 2nd Edition</td>
+ <td valign="top"><a href="using_samba/toc.html">Using Samba</a>, 2nd Edition</td>
  <td valign="top"><i>Using Samba</i>, Second Edition is a comprehensive guide to Samba administration. It covers all versions of Samba from 2.0 to 2.2, including selected features from an alpha version of 3.0, as well as the SWAT graphical configuration tool. Updated for Windows 2000, ME, and XP, the book also explores Samba's new role as a primary domain controller and domain member server, its support for the use of Windows NT/2000/XP authentication and filesystem security on the host Unix system, and accessing shared files and printers from Unix clients.</td>
 </tr>
 <tr>
- <td valign="top"><a href="manpages-3/index.html">Man pages</a></td>
+ <td valign="top"><a href="manpages/index.html">Man pages</a></td>
  <td valign="top">The Samba man pages in HTML.</td>
 </tr>
 <tr>
  <td valign="top"><a href="../../WHATSNEW.txt">WHATSNEW</a></td>
   <td valign="top">Samba Release Notes.</td>
 </tr>
-<tr>
- <td valign="top"><a href="../../README.VENDOR">README.VENDOR</a></td>
-  <td valign="top">VENDOR specific information.</td>
-</tr>
 </table></body></html>
